bitcampHomeworkWeekend05/src/ba/bitcamp/homeworkweekend05/task01/StoryElement.java

```diff
@@ -0,0 +1,31 @@
+package ba.bitcamp.homeworkweekend05.task01;
+
+public class StoryElement {
+
+	private String name;
+
+	public StoryElement(String name) {
+		super();
+		this.name = name;
+	}
+
+	public StoryElement() {
+		this("STElement");
+	}
+
+	// Getters and setters
+	public String getName() {
+		return name;
+	}
+
+	public void setName(String name) {
+		this.name = name;
+	}
+
+	// Basic toString method
```

Comment:
Javadoc komentari se trebaju pisati iznad metoda umjesto obicnih komentara, ako ih vec pises. Dobro bi bilo da ti to predje u naviku sto prije. To su ipak komentari koji se vide javno, tj. sluze za formiranje dokumentacije o projektu (i detaljniji opis onome ko poziva metodu).